recipes/fetchM/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "fetchM" %}
+{% set version = "0.1.1" %}
+{% set python_min = "3.7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/d2/f6/0a66917aa15b73ed962e613c399b1e537de1b1f5d8c2e81740968f83ce16/fetchm-0.1.1.tar.gz
```

Comment:
use the canonical url, with the appropriate cased version of the name as needed:
https://github.com/conda-forge/staged-recipes/blob/main/recipes/example/meta.yaml#L16